server/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -267,7 +285,7 @@ public void testGetIndexAnalyserWithoutIndexAnalyzers() throws IOException {
                     .analyzer("custom_analyzer")
                     .text("the qu1ck brown fox-dog"),
                 "text", null, null, registry, environment, maxTokenCount));
-        assertEquals(e.getMessage(), "failed to find global analyzer [custom_analyzer]");
+        assertEquals(e.getMessage(), "failed to find analyzer [custom_analyzer]");
```

Comment:
nit: might be worth fixing "expected" and "actual" arguments of assertEquals here while you are at it. The error message reads the wrong way round this way... seems to also be the wrong way round in several other places in this test.